make/common/templates/ui/env

```diff
@@ -5,6 +5,6 @@ JOBSERVICE_SECRET=$jobservice_secret
 GODEBUG=netdns=cgo
 ADMINSERVER_URL=$adminserver_url
 UAA_CA_ROOT=/etc/ui/certificates/uaa_ca.pem
-_REDIS_URL=$redis_url
+_REDIS_URL=$redis_host:$redis_port,100,$redis_password
```

Comment:
The db_index is set to default to 0?

Reply:
cannot set index value for beego, 0 is the hard coded.